js/src/3rdparty/etherscan/addressLink.js

```diff
@@ -14,8 +14,6 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-// links to chain explorers
-export const BASE_LINK_ACCOUNT_MORDEN = 'https://testnet.etherscan.io/address/';
-export const BASE_LINK_ACCOUNT_HOMESTEAD = 'https://etherscan.io/address/';
-export const BASE_LINK_TX_MORDEN = 'https://testnet.etherscan.io/tx/';
-export const BASE_LINK_TX_HOMESTEAD = 'https://etherscan.io/tx/';
+export default (address, isTestnet = false) => {
```

Comment:
Put txLint & addressLink functions into a single file, links.js (aligns with the rest and doesn't end up with 90% of the files have more boilerplate than actual code)